src/index.tsx

```diff
@@ -680,21 +680,15 @@ class App extends React.Component<{}, AppState> {
                   hitElement.isSelected = true;
                   // We duplicate the selected element if alt is pressed on Mouse down
                   if (e.altKey) {
-                    const element = newElement(
-                      hitElement.type,
-                      hitElement.x,
-                      hitElement.y,
-                      hitElement.strokeColor,
-                      hitElement.backgroundColor,
-                      hitElement.fillStyle,
-                      hitElement.strokeWidth,
-                      hitElement.roughness,
-                      hitElement.opacity,
-                      hitElement.width,
-                      hitElement.height
+                    elements.push(
+                      ...elements.reduce((duplicates, element) => {
+                        if (element.isSelected) {
+                          duplicates.push({ ...element });
+                          element.isSelected = false;
```

Comment:
missing generateDraw() command

Reply:
Also regenerating a new seed

Reply:
It should be copied, too. Works for me :thinking:

Reply:
![excalidraw_alt_drag_duplicate_texts](https://user-images.githubusercontent.com/5153846/72000131-f0f46400-3242-11ea-8857-a501b3f2e6b6.gif)

Reply:
> Also regenerating a new seed

ah.. and `id`.. Man. This has been a debacle :). Will fix after dinner. Can you still repro the text issue?